test/Conversion/amd/math-denorm-handling.mlir

```diff
@@ -35,3 +35,64 @@ module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 1 : i32, ttg.targ
     tt.return
   }
 }
+
+// -----
+
+#blocked = #ttg.blocked<{sizePerThread = [1], threadsPerWarp = [64], warpsPerCTA = [1], order = [0]}>
+module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 1 : i32, ttg.target = "hip:gfx942", "ttg.threads-per-warp" = 64 : i32} {
+  tt.func public @test_sqrt_f32(%arg0: tensor<64xf32, #blocked>) attributes {noinline = false} {
+    // LLVM_FTZ-LABEL: test_sqrt_f32
+    // LLVM_FTZ: llvm.amdgcn.sqrt.f32
+    //
+    // LLVM_NO_FTZ-LABEL: test_sqrt_f32
+    // LLVM_NO_FTZ: llvm.fcmp "ogt"
+    // LLVM_NO_FTZ: llvm.fmul
+    // LLVM_NO_FTZ-NEXT: llvm.select
+    // LLVM_NO_FTZ-NEXT: llvm.amdgcn.sqrt.f32
+    // LLVM_NO_FTZ: llvm.fmul
+    // LLVM_NO_FTZ-NEXT: llvm.select
+    %0 = math.sqrt %arg0 : tensor<64xf32, #blocked>
+    tt.return
+  }
+}
+
+// -----
+
+#blocked = #ttg.blocked<{sizePerThread = [1], threadsPerWarp = [64], warpsPerCTA = [1], order = [0]}>
+module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 1 : i32, ttg.target = "hip:gfx942", "ttg.threads-per-warp" = 64 : i32} {
+  tt.func public @test_sqrt_rn_f32(%arg0: tensor<64xf32, #blocked>) attributes {noinline = false} {
+    // LLVM_FTZ-LABEL: test_sqrt_rn_f32
+    // LLVM_FTZ: llvm.amdgcn.rsq.f32
+    // LLVM_FTZ: llvm.fmul
+    // LLVM_FTZ: llvm.fmul
+    // LLVM_FTZ: llvm.fneg
+    // LLVM_FTZ: llvm.intr.fma
+    // LLVM_FTZ-NEXT: llvm.intr.fma
+    // LLVM_FTZ-NEXT: llvm.intr.fma
+    // LLVM_FTZ-NEXT: llvm.fneg
+    // LLVM_FTZ-NEXT: llvm.intr.fma
+    // LLVM_FTZ-NEXT: llvm.intr.fma
+    // LLVM_FTZ-NEXT: llvm.intr.is.fpclass
+    // LLVM_FTZ-NEXT: llvm.select
+    //
+    // LLVM_NO_FTZ-LABEL: test_sqrt_rn_f32
+    // LLVM_NO_FTZ: llvm.intr.sqrt
+    %0 = tt.precise_sqrt %arg0 : tensor<64xf32, #blocked>
+    tt.return
+  }
+}
+
+// -----
+
+#blocked = #ttg.blocked<{sizePerThread = [1], threadsPerWarp = [64], warpsPerCTA = [1], order = [0]}>
+module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 1 : i32, ttg.target = "hip:gfx942", "ttg.threads-per-warp" = 64 : i32} {
+  tt.func public @test_sqrt_rn_f64(%arg0: tensor<64xf64, #blocked>) attributes {noinline = false} {
+    // LLVM_FTZ-LABEL: test_sqrt_rn_f64
```

Comment:
If both case are the same, you can define a `COMMON` prefix and then add `--check-prefix=COMMON,..` to both `RUN` commands.